wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,392 @@
+| ![][flag_CL] [Mathi](https://osu.ppy.sh/users/5339515) | 213 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 196 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 172 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 144 |
+| ![][flag_US] [idke](https://osu.ppy.sh/users/4650315) | 137 |
+| ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150) | 77 |
+| ![][flag_KR] [FlyingTuna](https://osu.ppy.sh/users/9224078) | 59 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 59 |
+| ![][flag_KR] [sayonara-bye](https://osu.ppy.sh/users/713266) | 58 |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 51 |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 44 |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 39 |
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 37 |
+| ![][flag_KR] [Reimu-Desu](https://osu.ppy.sh/users/948713) | 35 |
+| ![][flag_KR] [firebat92](https://osu.ppy.sh/users/1777162) | 34 |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | 33 |
+| ![][flag_US] [geckogates](https://osu.ppy.sh/users/252524) | 33 |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 27 |
+| ![][flag_KR] [Sure](https://osu.ppy.sh/users/70730) | 26 |
+| ![][flag_DE] [ShadowSoul](https://osu.ppy.sh/users/494970) | 20 |
+| ![][flag_US] [Freddie Benson](https://osu.ppy.sh/users/7342622) | 13 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 13 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 8 |
+| ![][flag_KR] [La Valse](https://osu.ppy.sh/users/70863) | 7 |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | 6 |
+| ![][flag_TW] [f0012979](https://osu.ppy.sh/users/12143) | 4 |
+| ![][flag_CN] [5231_Kinoko](https://osu.ppy.sh/users/181057) | 1 |
+| ![][flag_FI] [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+
+## Longest individual reigns
+
+*Note: Only reigns over 100 days are included. The current Rank #1 reign is excluded.*
+
+The following is a list of the longest consecutive days a player from the previous lists has held the Rank #1 spot (as of 3 June 2021).
+
+| Player | Reign Length (Days) |
+| --: | :-: |
+| ![][flag_DE] [WhiteCat](https://osu.ppy.sh/users/4504101) | 550 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 455 |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 258 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 256 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 242 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 176 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 173 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 167 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 150 |
+| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 144 |
+| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 133 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 109 |
```

Comment:
I want to see if this would be a change that other people think is useful.

```suggestion
| Player | From | To | Reign Length (Days) |
| --: | :-- | :-- | :-: |
| ![][flag_DE] [WhiteCat](https://osu.ppy.sh/users/4504101) | 2019-10-06 | 2021-04-08 | 550 |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2016-10-23 | 2018-01-21 | 455 |
| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-23 | 2008-09-06 | 258 |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2013-01-21 | 2013-10-04 | 256 |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-03-25 | 2015-11-22 | 242 |
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2012-06-05 | 2012-11-28 | 176 |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2014-08-17 | 2015-02-06 | 173 |
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2011-11-05 | 2012-04-20 | 167 |
| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2011-06-08 | 2011-11-05 | 150 |
| ![][flag_JP] [rrtyui](https://osu.ppy.sh/users/352328) | 2014-01-27 | 2014-06-20 | 144 |
| ![][flag_TW] [hvick225](https://osu.ppy.sh/users/50265) | 2015-12-31 | 2016-05-12 | 133 |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-10-03 | 2011-01-20 | 109 |
```

Reply:
I like that idea, gives a better overview

Reply:
could be a useful information to include i guess